### ISSUE TITLE
Security of systemd unit

### ISSUE THREAD

**Issue by leonardder** [jmdaweb/NVDARemoteServer#14]
In the current situation, the NVDA Remote server is running as root by default, which is a huge security risk and should be avoided. Packages should create a separate user for the server and the systemd unit should be changed in order to start the server with that specified user.

**Comment by jmdaweb** [comment]
I know that running daemons as an unprivileged user is a good practice. It should be changed for all linux distributions, not only systemd, but it is a complex task. Did you discover any vulnerability in the code which could be mitigated running as a standard user? This server is very simple and doesn't work with the file system, such as Apache or MySQL do. That is the reason because I haven't made this change before.

**Comment by leonardder** [comment]
[**@jmdaweb**](https://github.com/jmdaweb) commented on [10 okt. 2017 09:44 CEST](https://github.com/jmdaweb/NVDARemoteServer/issues/14#issuecomment-335389000 "2017-10-10T07:44:31Z - Replied by Github Reply Comments"):
> Did you discover any vulnerability in the code which could be mitigated running as a standard user?

Not yet, but even when you are pretty sure there are no vulnerabilities, as a system administrator I'm not quite happy to take the risk of running a process as root which really should not run as such.

**Comment by pvagner** [comment]
Incidentally I've just submitted the pull request #15 with my changes. I'm running the NVDARemoteServer on arch linux like this at least since 1.4 version now.

**Comment by jmdaweb** [comment]
Hello.
I have made some commits to solve this issue. Can you test and let me know if it works? I only have tested the Debian 8 version, but I have worked also in Debian 7 and RHL packages.
